usr/include/pkcs11types.h

```diff
@@ -36,8 +27,8 @@
 extern "C" {
 #endif
 
-#define CK_TRUE  (unsigned char)1
-#define CK_FALSE (unsigned char)0
+#define CK_TRUE  1
```

Comment:
same here, instead of reverting the change in this commit, update the original commit to not make the change at all.